scripts/quests/sandoria/RDM_AF3_Peace_for_the_Spirit.lua

```diff
@@ -0,0 +1,236 @@
+-----------------------------------
+-- Peace for the Spirit
+-----------------------------------
+-- Log ID: 0, Quest ID: 86
+-- Curilla !pos 27 0.1 0.1 233
+-- Sharzalion !pos 95 0 111 230
+-- Daggo !pos 89 1 119 230
+-- Miser Murphy !pos -14 -16 70 204
+-- Dry Fountain !pos -14 -16 70 204
+-- Oaken Box !pos -164 0 224 200
+-----------------------------------
+local ID = zones[xi.zone.FEIYIN]
```

Comment:
Can you make this `feiyinID` to keep consistency throughout the files.

Reply:
That is a great catch, Frank. I have updated to ensure consistency. For others, I will update as well in the future. Thanks!